tests/jerry/regression-test-issue-2230.js

```diff
@@ -0,0 +1,26 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+try {
+  ((new RegExp("[\\u0")).exec("u"));
+} catch (e) {
```

Comment:
One more note: is it expected that the regex exec throws an exception? In that case we should add an `assert (false)` after the regex and before the catch IMO. Otherwise the test could pass without checking any assertions.

(same note to all try-catches in both tests)

Reply:
Good catch! Also the PR is rebased to the latest master.